### PR TITLE
Passing sql file paths

### PR DESCRIPTION
**Pull request by MaximTar**
Tried to take into account your comments in the https://github.com/bryzgaloff/airflow-clickhouse-plugin/pull/41
(Besides, fixed badge in the README and patched workflow)

### CODE REVIEW

File: tests/unit/test_operator.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 import unittest
+from tempfile import NamedTemporaryFile
 from unittest import mock
 
+from airflow.models import DAG
+
 from airflow_clickhouse_plugin.hooks.clickhouse_hook import ClickHouseHook
 from airflow_clickhouse_plugin.operators.clickhouse_operator import ClickHouseOperator
 
+DEFAULT_DATE = datetime.datetime.now()
```
Comment (bryzgaloff) [inline tests/unit/test_operator.py:12]:
```suggestion
_TEST_START_DATE = datetime.datetime.now()
```

Comment (bryzgaloff) [reply tests/unit/test_operator.py:12]: Looks more descriptive to me

File: tests/unit/test_operator.py [hunk 2]
```diff
@@ -40,6 +47,39 @@ def test_defaults(self, clickhouse_hook_mock: mock.MagicMock):
...
+        with NamedTemporaryFile(suffix='.sql') as f:
+            f.write(b'{{ ds }}')
+            f.flush()
+            template_dir = os.path.dirname(f.name)
+            template_file = os.path.basename(f.name)
+
+            with DAG('test-dag', start_date=DEFAULT_DATE, template_searchpath=template_dir):
+                task = ClickHouseOperator(task_id='test_task', sql='SELECT 1')
+
+            task._sql = template_file
```
Comment (bryzgaloff) [inline tests/unit/test_operator.py:63]: Why do you override `task._sql` here and not simply set `sql` to `template_file` in the `ClickHouseOperator` constructor? I am not quite familiar with files functionality in Airflow, and I would assume `ClickHouseOperator(task_id='test_task', sql=template_file)` to work fine.

File: tests/unit/test_operator.py [hunk 3]
```diff
@@ -40,6 +47,39 @@ def test_defaults(self, clickhouse_hook_mock: mock.MagicMock):
         )
         clickhouse_hook_mock().run.assert_called_once_with(sql, None)
 
+    def test_template_fields_overrides(self):
+        assert ClickHouseOperator.template_fields == ('_sql',)
+
+    def test_resolve_template_files_value(self):
+        with NamedTemporaryFile(suffix='.sql') as f:
```
Comment (bryzgaloff) [inline tests/unit/test_operator.py:54]:
```suggestion
        with NamedTemporaryFile(suffix='.sql') as sql_file:
```

File: tests/unit/test_operator.py [hunk 4]
```diff
@@ -40,6 +47,39 @@ def test_defaults(self, clickhouse_hook_mock: mock.MagicMock):
...
+            f.write(b'{{ ds }}')
+            f.flush()
+            template_dir = os.path.dirname(f.name)
+            template_file = os.path.basename(f.name)
```
Comment (bryzgaloff) [inline tests/unit/test_operator.py:58]:
```suggestion
            sql_file_dir = os.path.dirname(sql_file.name)
            sql_file_name = os.path.basename(sql_file.name)
```

Comment (bryzgaloff) [reply tests/unit/test_operator.py:58]: Please apply the same changes to other tests too

**Review by bryzgaloff — CHANGES_REQUESTED** [review]
Thank you for your contribution! Have some questions and suggestions, please request my review again once done.

**Review by bryzgaloff — APPROVED** [review]
Well done, thank you! I will include this in the next release 👍

**Comment by bryzgaloff** [comment]
Also, adding you as a contributor, of course

**Comment by MaximTar** [comment]
> Also, adding you as a contributor, of course

Thanks, happy to help!
I hope you will have time to look at #63 also

**Comment by bryzgaloff** [comment]
Hey @MaximTar I wanted to publish this PR's update as the next version using your awesome #63 work, but looks like we need som extra changes to #63. It this functionality (SQL files paths) urgent for you? I may publish it manually, if required, or we may wait till the publishing PR is ready to test it right away :)

**Comment by darkebe** [comment]
Hello, thanks for this PR @MaximTar 
Do you know when this will be release ?

**Comment by bryzgaloff** [comment]
Hi @darkebe I was expecting #63 to be implemented to deploy this feature, but looks like we are stuck with the automated publishing to PyPI. Thus, I plan to publish the feature manually till early next week.

**Comment by MaximTar** [comment]
> 

I apologize that I disappeared - a blockage formed on all fronts, from which I just started to get out
now back to my request

**Comment by bryzgaloff** [comment]
No problem, your participation is highly appreciated! :)

I have just published this version manually as v0.10.1: https://github.com/bryzgaloff/airflow-clickhouse-plugin/releases/tag/v0.10.1

CC @darkebe @MaximTar

**Pull request merged**